clean.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+
+var deleteFolderRecursive = function(path) {
```

Comment:
I would move this file into `bin` (I realize it's not a binary file, but that's where I'm keeping executable scripts), and leave a comment explaining what it does and why it's needed.